services/src/projects/project.rs

```diff
@@ -583,9 +582,10 @@ mod tests {
                         "legend": false,
                     },
                     "symbology": {
-                        "raster": {
-                            "opacity": 1.0,
-                            "colorizer": "rgba"
+                        "type": "raster",
```

Comment:
So we differentiate between raster for raster and polygon/point/line for vector?
Maybe this is okay but what is your rational behind it?

Reply:
it was inconvient to have symbology -> vectorSymbology -> pointSymbology.